API.md

```diff
@@ -66,37 +67,377 @@ const category = {
 }
 ```
 
-API:
+# API:
 
+## /categories/
+
+```
+[{
+	id:string,
+	name:string,
+	homeImage:URL,
+	categoryImage:URL,
+	title:string,
+	subtitle:string,
+	brands:{[brandId]:brand}
+}]
+```
+
+## /categories/:id
+
+```
+{
+	id:string,
+	name:string,
+	homeImage:URL,
+	categoryImage:URL,
+	title:string,
+	subtitle:string,
+	brands:{[brandId]:brand}
+}
+```
+
+## /brands/
+
+```
+[{
+	id:string,
+	name:string,
+	image:URL
+}]
+```
+
+## /brands/:id
+
+```
+{
+	id:string,
+	name:string,
+	image:URL
+}
+```
+
+## /users/
```

Comment:
Getting all users with their reviews, orders, and addresses would end up with a large data set, which could be inefficient. Please clarify your requirements here to optimize our API response accordingly.